tests/unit/dojson/test_dojson_experiments.py

```diff
@@ -151,26 +151,65 @@ def test_experiment_names_and_affiliation_from_marcxml_119():
 
     result = strip_empty_values(experiments.do(create_record(snippet)))
 
-    assert result['affiliation'][0] == 'CERN'
+    assert result['affiliation'][0]['name'] == 'CERN'
+    assert result['affiliation'][0]['curated_relation'] == False
     assert result['experiment_names'][0]['title'] == 'CERN-ALPHA'
 
 
-def test_experiment_names_and_affiliation_from_marcxml_119_two_u():
```

Comment:
The same record is being tested few lines below in `test_experiment_names_and_affiliation_from_marcxml_multiple_119` which contains also fields `119__z`.